tests/backend/test_utils_fit_score.py

```diff
@@ -0,0 +1,174 @@
+Collaborated with product teams to create user-friendly, responsive interfaces.
+Automated test coverage with PyTest and Jest, resulting in 25% fewer production bugs.
+Deployed applications to AWS using Docker and managed CI/CD pipelines for faster releases.
+Frontend Developer Intern`
+Innovate Labs, Brooklyn, NY | Jan 2020 – May 2020
+
+Built dynamic and responsive components using React and Redux for a customer-facing web platform.
+Enhanced UI/UX design by integrating Tailwind CSS, boosting customer satisfaction by 20%.
+Worked closely with backend developers to consume REST APIs efficiently.
+Backend Developer Intern
+NextGen Tech, Jersey City, NJ | May 2019 – Aug 2019
+
+Developed RESTful API endpoints using Flask to support a large-scale ecommerce platform.
+Implemented database models in PostgreSQL, optimizing query performance by 15%.
+Created detailed API documentation to facilitate seamless frontend-backend collaboration.
+Projects
+TaskFlow (Personal Project)
+
+Developed a task management web app with Python (Flask) and React, supporting real-time updates using WebSockets.
+Integrated user authentication and role-based access control with JWT.
+RecipeMaster (Team Project)
+
+Built a recipe-sharing platform using Django and React, including features like search, filters, and user reviews.
+Deployed the application on AWS and maintained a CI/CD pipeline with GitHub Actions.
+Certifications
+AWS Certified Developer – Associate
+React Nanodegree – Udacity
+Professional Development
+Regular attendee at NYC Python and React Meetups.
+Contributor to open-source projects on GitHub, focusing on web development tools and frameworks.
+References available upon request.
+"""
+BAD_RESUME="""
+Jane Smith
+Los Angeles, CA | jane.smith@email.com | (555) 123-4567
+
+Objective
+Looking for any job that pays well, preferably with minimal effort.
+
+Education
+High School Diploma
+Springfield High School, Springfield, USA | Graduation Date: June 2016
+
+Technical Skills
+Proficient in Microsoft Word and Excel
+Typing speed of 50 words per minute
+Basic internet browsing skills
+Professional Experience
+Cashier
+Burger Town, Los Angeles, CA | May 2018 – Aug 2020
+
+Handled cash and credit card transactions.
+Restocked napkins and condiments.
+Cleaned tables and counters.
+Customer Service Representative
+CallTime Solutions, Los Angeles, CA | Sep 2020 – Present
+
+Answered customer calls about billing issues.
+Used a company-provided script to assist customers.
+Escalated complex issues to supervisors.
+Projects
+DIY Garden Blog (Personal Hobby)
+
+Started a blog about growing houseplants.
+Posted twice in 2021.
+Photography Portfolio (Unfinished)
+
+Took random pictures on my phone to post on social media.
+Certifications
+None
+Professional Development
+Attended one-day seminar on "Effective Communication in the Workplace" in 2019.
+"""
+def test_calculate_fit_score_normal():
```

Comment:
need to write documentation for all of these test functions

Reply:
fixed. TY